packages/components/src/DataList/components/DataListTags/DataListTags.css

```diff
@@ -1,27 +1,27 @@
+.tagWrapper {
+  position: relative;
+}
+
 .tags {
   display: flex;
-  position: relative;
   z-index: var(--elevation-default);
   overflow: hidden;
   gap: var(--space-small);
   white-space: nowrap;
+  mask-image: linear-gradient(
+    to left,
+    transparent,
+    transparent var(--space-large),
```

Comment:
Setting the stops using space values instead of the more common "%" because we want to ensure that the tags are fully clipped beneath the tag count. Same approach as the previous gradient with some adjustments.